ground/src/main/java/com/google/android/ground/ui/home/HomeScreenViewModel.kt

```diff
@@ -64,6 +64,7 @@ internal constructor(
       navigator.navigate(
         HomeScreenFragmentDirections.actionHomeScreenFragmentToDataCollectionFragment(
           draft.loiId,
+          draft.loiName ?: "",
```

Comment:
For consistency with other args, we should use `null` as the default value.

Reply:
I was getting a null pointer exception, which I tried to solve upstream here, but I realized that I should have been doing it downstream instead. Done.